deploy/chart/templates/0000_50_olm_16-packageserver.deployment.yaml

```diff
@@ -0,0 +1,63 @@
+{{ if eq .Values.installType "ocp" }}
```

Comment:
Couldn't we use a named partial template for the deployment (`_packageserver.deployment.yaml`) and include it this file and `_packageserver.clusterserviceversion.yaml`? This way we can keep the deployment definition in sync automatically. Although, since we're not expecting to use this for too long, and the packageserver doesn't change very often, maybe just having the note here will be fine.

Reply:
I'll go ahead and fix this.